packages/docs/src/helpers/SortObjectByPropNamesHelper.ts

```diff
@@ -0,0 +1,9 @@
+export const sortObjectByPropNames = (obj: any) => {
+  const arrFromObj = Object.entries(obj);
+
+  const sortedArr = arrFromObj.sort((element1: any, element2: any) => {
+    return element1[0].localeCompare(element2[0]);
+  });
+  const sortedObj = Object.fromEntries(sortedArr);
+  return sortedObj;
```

Comment:
Please, describe why you think it should be better to create a new Object instead of modifying the initial one in-place. I'm not saying that it's wrong, I'm just wondering if in-place modification could be more appropriate here.